docs/docs/Deployment/deployment-kubernetes.md

```diff
@@ -1,34 +1,30 @@
 ---
-title: Kubernetes
+title: Deploy Langflow on Kubernetes
 slug: /deployment-kubernetes
 ---
 
 
+This guide demonstrates deploying Langflow on a Kubernetes cluster. Two charts are available at the [Langflow Helm Charts repository](https://github.com/langflow-ai/langflow-helm-charts):
 
-This guide will help you get LangFlow up and running in Kubernetes cluster, including the following steps:
+- Install [Langflow as an IDE](deployment-kubernetes#install-the-langflow-ide-helm-chart) in a Kubernetes cluster for the complete Langflow development environment.
+- Install [Langflow as a standalone application](/deployment-kubernetes#install-the-langflow-runtime-helm-chart) to deploy a Langflow application in a more secure and stable environment.
 
-- Install [LangFlow as IDE](/deployment-kubernetes) in a Kubernetes cluster (for development)
-- Install [LangFlow as a standalone application](/deployment-kubernetes) in a Kubernetes cluster (for production runtime workloads)
+## Langflow IDE deployment
 
-## LangFlow (IDE) {#cb60b2f34e70490faf231cb0fe1a4b42}
+The Langflow IDE deployment is a complete environment for developers to create, test, and debug their flows. It includes both the API and the UI.
```

Comment:
The language is inconsistent. You're using "Install Langflow as an IDE" and "Langflow IDE deployment". It's not clear. Same for "Install Langflow as a standalone application" and "Langflow runtime deployment".